### PR TITLE
Add hidden option to toolbox

### PR DESCRIPTION
**Pull request by hata6502**
https://github.com/codex-team/editor.js/issues/1221
Screenshot from 2020-08-01 00-19-25

### CODE REVIEW

**Comment by hata6502** [comment]
Fixed to  allow false as toolbox config property value. 

Screenshot from 2020-08-01 00-19-25

File: src/components/modules/toolbar/toolbox.ts [hunk 1]
```diff
@@ -183,12 +183,16 @@ export default class Toolbox extends Module {
...
+
+    if ((userToolboxSettings ?? toolToolboxSettings) === false) {
+      return;
+    }
 
     const button = $.make('li', [ this.CSS.toolboxButton ]);
 
     button.dataset.tool = toolName;
-    button.innerHTML = userToolboxSettings.icon || toolToolboxSettings.icon;
+    button.innerHTML = (userToolboxSettings && userToolboxSettings.icon) || toolToolboxSettings.icon;
```
Comment (gohabereg) [inline src/components/modules/toolbar/toolbox.ts:195]:
```suggestion
    button.innerHTML = userToolboxSettings?.icon || toolToolboxSettings.icon;
```

Comment (gohabereg) [reply src/components/modules/toolbar/toolbox.ts:195]: My bad, && chaining is correct

**Review by gohabereg — APPROVED** [review]
Please, pull `next` branch, rebuild and add changelog. I can't do it as don't have permissions

File: types/tools/tool-settings.d.ts [hunk 2]
```diff
@@ -45,5 +45,5 @@ export interface ToolSettings {
   /**
    * Tool's Toolbox settings
```
Comment (neSpecc) [inline types/tools/tool-settings.d.ts:46]: Can you describe in jsdoc, what `false` means

File: src/components/modules/toolbar/toolbox.ts [hunk 3]
```diff
@@ -183,12 +183,16 @@ export default class Toolbox extends Module {
     //   return;
     // }
 
-    const userToolboxSettings = this.Editor.Tools.getToolSettings(toolName)[userSettings.TOOLBOX] || {};
+    const userToolboxSettings = this.Editor.Tools.getToolSettings(toolName)[userSettings.TOOLBOX];
+
+    if ((userToolboxSettings ?? toolToolboxSettings) === false) {
```
Comment (neSpecc) [inline src/components/modules/toolbar/toolbox.ts:188]: Pls add a jsdoc comment describing this moment

**Comment by neSpecc** [comment]
and pls open write access for us

**Comment by hata6502** [comment]
> and pls open write access for us

Sorry, I use GitHub organization, so I can't allow edits from maintainers. 
https://qiita.com/ryysud/items/4d31dd86efab34a7fbaf#github-%E3%82%B5%E3%83%9D%E3%83%BC%E3%83%88%E3%81%8B%E3%82%89%E3%81%AE%E5%9B%9E%E7%AD%94

File: types/tools/tool-settings.d.ts (hunk 2 above)
Comment (hata6502) [reply types/tools/tool-settings.d.ts:46]: Fixed.

File: src/components/modules/toolbar/toolbox.ts (hunk 3 above)
Comment (hata6502) [reply src/components/modules/toolbar/toolbox.ts:191]: Fixed.

**Comment by hata6502** [comment]
> Please, pull `next` branch, rebuild and add changelog. I can't do it as don't have permissions

Fixed at: 
https://github.com/codex-team/editor.js/pull/1220/commits/4bba5e72c65db1b2ba4d53eaff864ee4d6278549%20https://github.com/codex-team/editor.js/pull/1220/commits/6e0d92e796894bbff326d53fc70a8b5e310e16c8

**Review by gohabereg — APPROVED** [review]

**Review by neSpecc — APPROVED** [review]

**Review by khaydarov — APPROVED** [review]

**Pull request merged**